web-ui/src/backup_account/page.spec.js

```diff
@@ -41,5 +42,29 @@ describe('BackupAccount', () => {
       pageInstance.saveBackupEmail('success');
       expect(page.find(Confirmation).length).toEqual(1);
     });
+
+    context('on submit error', () => {
+      beforeEach(() => {
+        pageInstance.saveBackupEmail('error');
+      });
+
+      it('returns snackbar component on error', () => {
+        const snackbar = pageInstance.showSnackbarOnError(pageInstance.props.t);
+        expect(snackbar).toEqual(<SnackbarNotification message='backup-account.error.submit-error' isError />);
+      });
+
+      it('returns nothing when there is no error', () => {
+        pageInstance.saveBackupEmail('success');
+        const snackbar = pageInstance.showSnackbarOnError(pageInstance.props.t);
+        expect(snackbar).toEqual(undefined);
+      });
+
+      it('renders snackbar notification on error', () => {
+        const snackbar = page.find(SnackbarNotification);
+        expect(snackbar).toExist();
+        expect(snackbar.props().message).toEqual('backup-account.error.submit-error');
+        expect(snackbar.props().isError).toEqual(true);
+      });
```

Comment:
So the 'returns snackbar ...' test is testing the function that returns the component whereas the second one 'renders snackbar ...' is testing that the component exists on the page when everything's rendered. Does the difference make sense?